src/renderer/viz/expressions.js

```diff
@@ -120,9 +137,291 @@
  * More information in {@link carto.expressions.palettes|carto.expressions.palettes}.
  *
  * @typedef {} Palette
+ * @name Palette
+ * @memberof Types
+ * @api
+ */
+
+/**
+ * Type of Image Expressions.
+ *
+ * More information in {@link carto.expressions.image|carto.expressions.image}.
+ *
+ * @typedef {} Image
+ * @name Image
+ * @memberof Types
  * @api
  */
 
+/**
+ * Type of Translate Expressions.
```

Comment:
Transformation type can be a translation, a rotation, or a chain of those two.

Reply:
Adding this also to the `Viz`.